docs/source/getting_started_guide/trying_sample_commands.rst

```diff
@@ -0,0 +1,45 @@
+###################
+Try Sample Commands
```

Comment:
A quick note near the top to let the use know that for these commands to work, they need to have set up the 3 node network listed in your other guide.

Reply:
Done